src/main/java/ch/njol/skript/util/slot/EquipmentSlot.java

```diff
@@ -110,6 +110,19 @@ public ItemStack get(final EntityEquipment e) {
 			public void set(final EntityEquipment e, final @Nullable ItemStack item) {
 				e.setBoots(item);
 			}
+		},
+
+		BODY() {
+			@Override
+			@Nullable
+			public ItemStack get(final EntityEquipment e) {
+				return e.getItem(org.bukkit.inventory.EquipmentSlot.BODY);
+			}
+
+			@Override
+			public void set(final EntityEquipment e, final @Nullable ItemStack item) {
+				e.setItem(org.bukkit.inventory.EquipmentSlot.BODY, item);
```

Comment:
```suggestion
				e.setItem(org.bukkit.inventory.EquipmentSlot.BODY, item);
```
shouldn't this be imported?

Reply:
With the name of the class being the same it errors out

Reply:
fair